tutorials/sim/plot_SimulateCombined.py

```diff
@@ -152,3 +153,41 @@
 
 # Plot the simulated data, in the time domain
 plot_time_series(times, sig)
+
+###################################################################################################
+# Simulate Peak Oscillation
+# -------------------------
+#
+# Next, we will simulate time series with a peak in the power spectrum, that we can
+# define in terms of the specific location and shape of the oscillatory peak.
+#
+# In order to make this simulation, we precompute an aperiodic signal, to which
+# we can add an oscillatory component to make the overall signal.
+#
+# To do so, we use the :func:`~.sim_peak_oscillation` function to add an oscillation to
+# the aperiodic component, specifying a desired central frequency, bandwidth, and peak height.
+#
+
+###################################################################################################
+
+# Precompute an aperiodic time series
+ap_sig = sim_powerlaw(n_seconds, fs, exponent=-1)
+
+# Define settings that define the peak to add
+freq = 10
+bw = 3
+height = 1
```

Comment:
Maybe increase height so the oscillatory peak is more distinct? Or increase n_seconds so spectra is a bit more stable. I'll play around with this.